src/sc/sidechainrpc.cpp

```diff
@@ -8,13 +8,13 @@
 
 #include <wallet/wallet.h>
 
+#include <core_io.h>
+#include <rpc/server.h>
+#include <main.h>
+#include <init.h>
+
 extern UniValue ValueFromAmount(const CAmount& amount);
 extern CAmount AmountFromValue(const UniValue& value);
```

Comment:
Sorry I left out these lines in previous comment. You can remove them out as these declarations are in `rpc/server.h` already